content/en/news/releases/1.6.x/announcing-1.6/change-notes/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Change Notes
+description: Istio 1.6 release notes.
+weight: 10
+---
+
+## User facing changes
+- ***Added*** experimental support for the [Kubernetes Service APIs](https://github.com/kubernetes-sigs/service-apis).
+- ***Added*** the new [Workload Entry](https://preliminary.istio.io/docs/reference/config/networking/workload-entry/) resource. This will allow easier configuration for non-Kubernetes workloads to join the mesh.
+- ***Added*** support for configuring Gateway topology settings like number of trusted proxies deployed in front
+- ***Added*** experimental mesh-wide tracing configuration API. This API provides control of trace sampling rates, the maximum tag lengths for URL tags, and custom tags extraction for all traces within the mesh. 
+- ***Added*** experimental request classification filter support. This feature enables operators to configure new attributes for use in telemetry, based on request information. A primary use case for this feature is labeling of traffic by API method.  TODO: doc
+- ***Added*** Wasm extension distribution support. TODO: docs
+- TODO: add a statement around networking v1alpha3 API. Will it be removed in 1.6 or announce deprecation?
+
+## Improvements
+- ***Improved*** support for [Kubernetes Ingress](https://preliminary.istio.io/docs/tasks/traffic-management/ingress/kubernetes-ingress/), adding support for reading certificates from Secrets, `pathType`, and IngressClass.
+- ***Added*** a new `proxy.istio.io/config` annotation to override proxy configuration per pod.
```

Comment:
this is cool, I hope there is sample for this too!  Can I use this to customize preStop configuration per pod?

Reply:
@jacob-delgado did you complete a sample for the tracing experimental API. I am not sure I wish to torture myself with more documentation work, but I'm happy to help take some load off your plate so you can focus on the release notes. I Will need a gist to work with. Feel free to leave a message here if you wish to take me up on this offer. If not, I would recommend removing from the release notes. If we communicate a feature without documentation, our audience will ask "Where is the documentation?" and then we have to scramble to produce some.

Reply:
@sdake Thank you so much for the offer. John wrote up a similar example found https://gist.github.com/howardjohn/15618fd3a1d2ed54b95ae6fb50548433. It would be something similar to

```yaml
        annotations:
          proxy.istio.io/config: |
            tracing:
              custom_tags:
                tag_header:                    # user-defined name
                  header:
                    name: <CLIENT-HEADER>
                    defaultValue: <VALUE>      # optional
```

Reply:
@sdake Found an issue with the annotations, but I've put up a fix for istio 1.6.1. I'll add it to the known issues for now.

PR to resolve this is https://github.com/istio/istio.io/pull/7348. I will link to that when we ship 1.6.1.